Makefile

```
@@ -5,10 +5,33 @@ ifdef ANDROID_ARCH
 	endif
 endif
 
+
 # Default is build for arm
 ANDROID_FULL_ARCH?=armeabi-v7a
 GRADLE_TASK?=assembleArm
 
+# find the path where the SDK is installed
+ifdef SDK
+        ANDROID_SDK_FULLPATH?=$(SDK)
+else
+        ifdef ANDROID_HOME
+                ANDROID_SDK_FULLPATH?=$(ANDROID_HOME)
+        else
+                ANDROID_SDK_FULLPATH?=`realpath ../../../base/toolchain/android-sdk-linux`
```

Shouldn't that be something like:
```suggestion
                ANDROID_SDK_FULLPATH?=$(shell realpath ../../../base/toolchain/android-sdk-linux)
```